src/bun.js/bindings/bindings.zig

```diff
@@ -5474,7 +5467,7 @@ pub const JSValue = enum(i64) {
     /// per invocation.
     ///
     /// This function will eventually replace `get`.
```

Comment:
this comment can be updated, now that this function has replaced `get`. additionally the line in get_unsafe can be edited to mention `get` instead of `get2`